packages/twenty-front/src/modules/settings/data-model/fields/forms/date/components/SettingsDataModelFieldDateForm.tsx

```diff
@@ -30,27 +59,74 @@ export const SettingsDataModelFieldDateForm = ({
 }: SettingsDataModelFieldDateFormProps) => {
   const { t } = useLingui();
 
-  const { control } = useFormContext<SettingsDataModelFieldDateFormValues>();
+  const { control, watch } =
+    useFormContext<SettingsDataModelFieldDateFormValues>();
 
-  const { initialDisplayAsRelativeDateValue } =
+  const { initialDisplayFormat, initialCustomUnicode35DateFormat } =
     useDateSettingsFormInitialValues({
       fieldMetadataItem,
     });
 
+  const displayFormatFromForm = watch('settings.displayFormat');
+
+  const showCustomFormatTextInput = displayFormatFromForm
+    ? isDateFieldCustomDisplayFormat(displayFormatFromForm)
+    : isDateFieldCustomDisplayFormat(initialDisplayFormat);
+
   return (
-    <Controller
-      name="settings.displayAsRelativeDate"
-      control={control}
-      defaultValue={initialDisplayAsRelativeDateValue}
-      render={({ field: { onChange, value } }) => (
-        <SettingsOptionCardContentToggle
-          Icon={IconSlash}
-          title={t`Display as relative date`}
-          checked={value ?? false}
-          disabled={disabled}
-          onChange={onChange}
+    <>
+      <Controller
+        name="settings.displayFormat"
+        control={control}
+        defaultValue={initialDisplayFormat}
+        render={({ field: { onChange, value } }) => (
+          <SettingsOptionCardContentSelect
+            Icon={IconSlash}
+            title={t`Display Format`}
+            disabled={disabled}
+            description={t`Choose the format used to display date value`}
```

Comment:
Maybe  when the select format is custom we could update the description and instead a link to https://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table?
Like, description would be "Enter in Unicode format" with a link to the above anchor on "Unicode"